src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Get-AzsDiskMigrationJob.ps1

```diff
@@ -0,0 +1,173 @@
+Changes may cause incorrect behavior and will be lost if the code is regenerated.
+#>
+
+<#
+.SYNOPSIS
+    Get or list migration job status.
+
+.DESCRIPTION
+    Returns a list of disk migration jobs.
+
+.PARAMETER ResourceId
+    The resource id.
+
+.PARAMETER Status
+    The parameters of disk migration job status.
+
+.PARAMETER Location
+    Location of the resource.
+
+.PARAMETER Name
+    The migration job guid name.
+
+.PARAMETER InputObject
+    The input object of type Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob.
+
+.EXAMPLE
+
+    PS C:\> $migrations = Get-AzsDiskMigrationJob -location local
+    PS C:\> $migration = Get-AzsDiskMigrationJob -location local -Name $migrations[0].MigrationId
+#>
+function Get-AzsDiskMigrationJob
+{
+    [OutputType([Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob])]
+    [CmdletBinding(DefaultParameterSetName='DiskMigrationJobs_List')]
+    param(    
+        [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true, ParameterSetName = 'ResourceId_DiskMigrationJobs_Get')]
+        [System.String]
+        $ResourceId,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'DiskMigrationJobs_List')]
+        [System.String]
+        $Status,
+    
+        [Parameter(Mandatory = $false, ParameterSetName = 'DiskMigrationJobs_Get')]
+        [Parameter(Mandatory = $false, ParameterSetName = 'DiskMigrationJobs_List')]
+        [System.String]
+        $Location,
+    
+        [Parameter(Mandatory = $true, ParameterSetName = 'DiskMigrationJobs_Get')]
+        [Alias('MigrationId')]
+        [System.String]
+        $Name,
+    
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true, ParameterSetName = 'InputObject_DiskMigrationJobs_Get')]
+        [Microsoft.AzureStack.Management.Compute.Admin.Models.DiskMigrationJob]
+        $InputObject
+    )
+
+    Begin 
+    {
+	    Initialize-PSSwaggerDependencies -Azure
+        $tracerObject = $null
+        if (('continue' -eq $DebugPreference) -or ('inquire' -eq $DebugPreference)) {
+            $oldDebugPreference = $global:DebugPreference
+			$global:DebugPreference = "continue"
+            $tracerObject = New-PSSwaggerClientTracing
+            Register-PSSwaggerClientTracing -TracerObject $tracerObject
+        }
+	}
+
+    Process {
+    
+    $ErrorActionPreference = 'Stop'
+
+    $NewServiceClient_params = @{
+        FullClientTypeName = 'Microsoft.AzureStack.Management.Compute.Admin.ComputeAdminClient'
+    }
+
+    $GlobalParameterHashtable = @{}
+    $NewServiceClient_params['GlobalParameterHashtable'] = $GlobalParameterHashtable
+     
+    $GlobalParameterHashtable['SubscriptionId'] = $null
+    if($PSBoundParameters.ContainsKey('SubscriptionId')) {
+        $GlobalParameterHashtable['SubscriptionId'] = $PSBoundParameters['SubscriptionId']
+    }
+
+    $ComputeAdminClient = New-ServiceClient @NewServiceClient_params
+
+    $MigrationId = $Name
```

Comment:
Remove

Reply:
We'd keep what it is look like because it is autogerated. And I don't think it has issue.